templates/examples-webgpu.html

```diff
@@ -4,7 +4,11 @@
 These examples demonstrate how to use Bevy's features in a minimal, easy to understand way.
 Click an example below to run it in your browser (using WASM + WebGPU) and view the source code.
 You can also view these examples (and others) in the <a href="https://github.com/bevyengine/bevy/tree/main/examples#examples">Bevy repo</a>.
-<h2 class="warning">Support Warning</h2>
-WebGPU is currently only supported on Chrome starting with version 113, and only on desktop.
-If they don't work on your configuration, you can check the WebGL2 examples <a href="/examples">here</a>.
+<aside class="callout callout--warning">
+  <h3 class="warning">Support Warning</h3>
+  <p>
+    WebGPU is currently only supported on Chrome starting with version 113, and only on desktop.
+    If they don't work on your configuration, you can check the WebGL2 examples <a href="/examples">here</a>.
```

Comment:
for a followup PR: this is no longer true, WebGPU also works on some mobile browser, and on SafarI TP, and Firefox Nightly
We should probably link to something like https://caniuse.com/webgpu for a list or https://browserleaks.com/webgpu so that user can test their browser instead of keeping a list ourselves